src/main/resources/static/js/MainWindow.js

```diff
@@ -675,448 +761,479 @@ var initial = 1000;
-        },{
-            type: "info",
-            placement: {
-                from: "top",
-                align: "left"
-            },
-            animate: {
-                enter: "animated fadeInDown",
-                exit: "animated fadeOutUp"
-            }
-        });
-        return false;
-    } else if (streams.length === 0) {
-        $.notify({
-            message: "Please add a datastream before importing data."
-        },{
-            type: "info",
-            placement: {
-                from: "top",
-                align: "left"
-            },
-            animate: {
-                enter: "animated fadeInDown",
-                exit: "animated fadeOutUp"
-            }
-        });
-        return false;
-    } else {
-
-        $.notify({
-            message: "Import started"
-        },{
-            type: "info",
-            allow_dismiss: true,
-            placement: {
-                from: "top",
-                align: "left"
-            },
-            animate: {
-                enter: "animated fadeInDown",
-                exit: "animated fadeOutUp"
-            }
-        });
-        addToLog("Import of File " + name + " started.");
-        document.getElementById("progress").value = 0;
-        id = setInterval(progress, initial);
-        $.ajax({
-            type: "POST",
-            url: "importQueue",
-            data: mydata,
-            success: function (e) {
-                addToLog(e);
-                $.notify({
-                    message: "Import finished"
-                },{
-                    type: "success",
-                    allow_dismiss: true,
-                    placement: {
-                        from: "top",
-                        align: "left"
-                    },
-                    animate: {
-                        enter: "animated fadeInDown",
-                        exit: "animated fadeOutUp"
-                    }
-                });
-            },
-            error: function (e) {
-                $.notify({
-                    message: "Import failed. Check log for errors."
-                },{
-                    type: "danger",
-                    allow_dismiss: true,
-                    placement: {
-                        from: "top",
-                        align: "left"
-                    },
-                    animate: {
-                        enter: "animated fadeInDown",
-                        exit: "animated fadeOutUp"
-                    }
-                });
-                addToLog(e.responseText);
-                clearInterval(id);
-            }
-        });
-    }
+	var name;
+	if ($("input[name=source]:eq(0)").is(":checked")) {
+		name = $("#file").val().split("\\").pop();
+	} else if ($("input[name=source]:eq(1)").is(":checked")) {
+		name = $("#sourceinput").val().split("/").pop();
+	}
+
+	try {
+		new RegExp(currentDelimiter);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found non-literal argument to RegExp Constructor](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4781409)